VideoLocker/res/values/errors.xml

```diff
@@ -1,6 +1,9 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
 
+    <!-- An error occurred but we don't know why -->
+    <string name="error_unknown">Unknown error. Please try again later.</string>
```

Comment:
Could we have "network" in the string somewhere, is it's specifically related to that context.